src/crypto.js

```diff
@@ -1,10 +1,22 @@
 import { ERRORS, checkObject, MasqError } from './errors'
 
+const _checkEncodingFormat = (format) => {
+  if (format !== 'hex' && format !== 'base64') throw new MasqError(ERRORS.INVALID_ENCODING_FORMAT)
+}
+
 const genRandomBuffer = (len = 16) => {
   const values = window.crypto.getRandomValues(new Uint8Array(len))
   return Buffer.from(values)
 }
 
+const genRandomBufferAsStr = (len = 16, encodingFormat) => {
+  if (encodingFormat) {
+    _checkEncodingFormat(encodingFormat)
+  }
+  const values = window.crypto.getRandomValues(new Uint8Array(len))
+  return encodingFormat ? Buffer.from(values).toString(encodingFormat) : Buffer.from(values)
```

Comment:
Once you get the buffer with `genRandomBuffer`, return a string, an use a default encoding if none is specified